docs/context_standards/README.MD

```diff
@@ -253,4 +257,28 @@ The following is the format for a process.
     "Sibling": "LIST, Sibling process objects",
     "Child": "LIST, of Child process objects"
 }
+```
+
+## DBot Score
```

Comment:
you should switch between InfoFIle and DBotScore.

Reply:
You want the order reversed? Infofile on top then Dbot score?

Reply:
no, it seems that the title of DBot Score explains InfoFile and vice versa